models/gc_tiger_lb2/meta.json

```diff
@@ -0,0 +1,113 @@
+    "inputs": [
+      {
+        "label": "Whole-slide image",
+        "description": "H&E-stained whole-slide image of breast cancer histopathology",
+        "format": "DICOM",
+        "modality": "SM",
+        "bodypartexamined": "Breast",
+        "slicethickness": "",
+        "non-contrast": false,
+        "contrast": false
+      }
+    ],
+    "outputs": [
+      {
+        "type": "Prediction",
+        "valueType": "Probability",
+        "label": "TIL score",
+        "description": "Percentage of stromal area covered by tumour infiltrating lymphocytes. Values between 0 (percent) to 100 (percent).",
+        "classes": []
+      }
+    ],
+    "model": {
+      "architecture": "Combination of multiple U-Nets with EfficientNet B2/B0 encoders, and YOLOv5 networks for detection.",
+      "training": "supervised",
+      "cmpapproach": "2D"
+    },
+    "data": {
+      "training": {
+        "vol_samples": 230
+      },
+      "evaluation": {
+        "vol_samples": 58
+      },
+      "public": false,
+      "external": false
+    }
+  },
+  "details": {
+    "name": "LB2",
+    "version": "55c49c9e-4216-4142-b1c8-f5d85781add3",
+    "devteam": "VUNO",
+    "type": "Segmentation/Prediction hybrid",
+    "date": {
+      "weights": "2023-07-06",
+      "code": "2023-07-06",
+      "pub": "2022-08-26"
+    },
+    "cite": "",
+    "license": {
+      "code": "Apache 2.0",
+      "weights": "CC BY-NC 4.0"
+    },
+    "publications": [],
+    "github": "https://github.com/vuno/tiger_challenge",
+    "zenodo": "https://doi.org/10.5281/zenodo.8112147",
+    "colab": "",
+    "slicer": false
+  },
+  "info": {
+    "use": {
+      "title": "Intended use",
+      "text": "Prediction of the percentage of stomal area covered by tumour infiltrating lymphocytes on H&E-stained whole-slide image of breast cancer histopathology.",
+      "references": [],
+      "tables": []
+    },
+    "analyses": {
+      "title": "Evaluation",
+      "text": "The prognostic value of the automatic \"TIL score\" generated by the submitted algorithms were computed for the test set. This was done by building a multivariate Cox regression model trained with predefined clinical variables and the produced TILs score. The concordance index (Uno’s C-index) of this model was computed and the algorithms were ranked based on its value. ",
+      "references": [
+        {
+          "label": "On the C-statistics for Evaluating Overall Adequacy of Risk Prediction Procedures with Censored Survival Data",
+          "uri": "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3079915/"
+        }
+      ],
+      "tables": []
+    },
+    "evaluation": {
+      "title": "Evaluation data",
+      "text": "The test set consists of a separate dataset of n=707 H&E-stained whole-slide breast cancer histopathology images",
```

Comment:
Is the mentioned dataset closed-source? Please specify or provide a public reference.#
Why is evaluation `summary.data.evaluation.vol_samples = 58 << n = 707`?

Reply:
We will update this.

Reply:
This was changed to the correct value.